content/en/core/releases/3.14.0.md

```diff
@@ -0,0 +1,143 @@
+# 3.14.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### New dialog showing sync status
+
+When an offline user manually triggers a sync via the hamburger menu, a notification dialog is opened indicating the status of the sync process.
+
+Sync in progress:
+![](../images/3.14.0-5207-current-sync.png)
+
+Sync complete:
+![](../images/3.14.0-5207-sync-done.png)
+
+Sync failed (with retry option):
+![](../images/3.14.0-5207-sync-failed.png)
+
+[#5207](https://github.com/medic/cht-core/issues/5207): Show dialog when user clicks "Sync Now" in hamburger menu
+
+#### Updated icon for the Create Report button
+
+The icon for the Create Report button has been updated so that its purpose is more clear.
+
+Old Icon | New Icon
+:-------------------------:|:-------------------------:
+![](../images/3.14.0-6403-old.png)  |  ![](../images/3.14.0-6403-new.png)
+
+[#6403](https://github.com/medic/cht-core/issues/6403): Change the Create Report button icon to better indicate what it does
+
+#### Display all dates in Bikram Sambat format when Nepali locale selected
+
+For years, the CHT has supported recording the answers to `date` questions in forms using the [Bikram Sambat](https://en.wikipedia.org/wiki/Vikram_Samvat) calendar format when the user has selected the Nepali locale. However, the date values displayed throughout the app would still be shown in the [Gregorian](https://en.wikipedia.org/wiki/Gregorian_calendar) calendar format.
+
+Now the application has been updated to also use the Bikram Sambat format when displaying dates in places such as the contact summary.
+
+![](../images/3.14.0-7294-contact-summary.png)
```

Comment:
This screenshot has a bunch of untranslated titles which looks unprofessional. Also because my Nepali isn't fluent I can't obviously see what is a date and what isn't. Just like above can we do a before/after so we can see how the Gregorian date is now Bikram? You probably only need one or two of the fields in the contact summary rather than the whole thing.